Orange/widgets/utils/owlearnerwidget.py

```diff
@@ -123,7 +123,7 @@ def __init__(self):
         self.preprocessors = None
         self.outdated_settings = False
         self.setup_layout()
-        QTimer.singleShot(0, self.apply)
+        QTimer.singleShot(0, self.update_learner)
```

Comment:
I did try that. At the least that breaks 'Linear Regression' widget because it overrides the `add_bottom_buttons` method and does not 'define' `unconditional_apply` so it fails to even initialize.